packages/dd-trace/src/telemetry/send-data.js

```diff
@@ -16,6 +15,12 @@ function sendData (config, application, host, reqType, payload = {}) {
       'dd-telemetry-request-type': reqType
     }
   }
+  if (url) {
+    options.url = url
```

Comment:
Any downside of always setting all 3?

Reply:
Also, can this be moved to `request.js` so that UDS is supported for all requests in the first place?

Reply:
+1 on moving it to request.js as we'll keep copy pasting it otherwise, including in the upcoming remote config.